packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1667,7 +1667,7 @@ class RoundedRectSliderTrackShape extends SliderTrackShape with BaseSliderTrackS
       isDiscrete: isDiscrete,
     );
     final Radius trackRadius = Radius.circular(trackRect.height / 2);
-    final Radius activeTrackRadius = Radius.circular(trackRect.height / 2 + 1);
```

Comment:
Maybe that's not actually a bug :P -- `additionalActiveTrackHeight` is not exposed to the user, and thus the `activeTrackRadius` will always have that `+ 1`. It becomes a bug when the user tries to extend from `RoundedRectSliderTrackShape` and override the `paint` function with a different value for `additionalActiveTrackHeight`, am I right?

Reply:
yes, I think that is the better way to test this, also that seems to be the only way that this issue become a problem. In real live, one must override RoundedRectSliderTrackShape and provide a different additionalActiveTrackHeight in order for this to become a problem

Reply:
Exactly!